cypress/integration/tests/administrator/proxy/proxy.enable.disable.test.ts

```diff
@@ -22,6 +30,35 @@ describe("Proxy operations", () => {
         preservecookies();
     });
 
+    it("Http Proxy port and host field validation", function () {
+        Proxy.open();
+        selectCheckBox(ProxyViewSelectors.httpSwitch);
+        proxy.fillHost(ProxyType.http, proxy.hostname);
+        proxy.fillPort(ProxyType.http, "Invalid port");
+        cy.get(ProxyViewSelectors.portHelper).contains("This field is required");
+        cy.get(submitButton).should("be.disabled");
+        unSelectCheckBox(ProxyViewSelectors.httpSwitch);
```

Comment:
Can disableProxy method be used here?

Reply:
Not `disableProxy`, but for sure there is a more reusable approach.

Solved in bdcd68008f4c29413d50f2ffabf64a035a98d55d.